common/djangoapps/track/tests/test_middleware.py

```diff
@@ -60,6 +60,7 @@ def test_default_request_context(self):
             'session': '',
             'username': '',
             'ip': '127.0.0.1',
+            'remote_addr': '127.0.0.1',
```

Comment:
Just making the least invasive change here.  In this case `ip` and `remote_addr` are the same and the test case already uses `127.0.0.1`.